### PR TITLE
Container handler

### PR DESCRIPTION
**Pull request by rcohencyberarmor**
## Overview

container handling implementation:
we are going to monitor any new containers and collect realtime information for filter the SBOM

### CODE REVIEW

**Comment by matthyx** [comment]
@rcohencyberarmor can you rebase now that your other PR has merged?

File: pkg/config/config.go [hunk 1]
```diff
@@ -9,16 +9,17 @@ import (
...
+	CONFIG_ENV_VAR   = "CONFIG_ENV_VAR"
 )
 
 type Config struct {
 	data ConfigDataInterface
 }
 
 func (cfg *Config) getConfigFilePath() (string, bool) {
-	return os.LookupEnv("SNIFFER_CONFIG")
+	return os.LookupEnv(CONFIG_ENV_VAR)
```
Comment (matthyx) [inline pkg/config/config.go:22]: are you sure it was rebased correctly? I think this was fixed in your previous PR?

**Review by matthyx — CHANGES_REQUESTED** [review]
it seems the rebase didn't work, or your previous PR didn't merge, I still see snake case

**Comment by dwertent** [comment]
This was not rebased well.

**Comment by dwertent** [comment]
Please open a new branch from the main branch.

**Pull request closed without merging**